instance.go

```diff
@@ -146,28 +145,28 @@ func (i *Instance) UpdateFromQList(qlist string) (err error) {
 	return nil
 }
 
-// CacheDat holds information that pertains an existing ISC database
-type CacheDat struct {
+// ISCDat holds information that pertains an existing ISC database
+type ISCDat struct {
 	Path       string
 	Permission string
 	Owner      string
 	Group      string
 	Exists     bool
 }
 
-// DetermineCacheDatInfo will parse the ensemble instance's CPF file for its databases (CACHE.DAT).
-// It will get the path of the CACHE.DAT file, the permissions on it, and its owning user / group.
-// The function returns a map of cacheDat structs containing the above information using the name of the database as its key.
-func (i *Instance) DetermineCacheDatInfo() (map[string]CacheDat, error) {
+// DetermineISCDatInfo will parse the ensemble instance's CPF file for its databases (CACHE.DAT, IRIS.DAT).
```

Comment:
Since this is already an API break, I think that I would just drop the `ISC`/`isc` from the type, function, and variable names. It's implied by the elements existing in the context of the library.